apps/ledger-live-desktop/src/newArch/features/Collectibles/Nfts/BreadCrumb/useBreadCrumbModel.tsx

```diff
@@ -26,6 +26,7 @@ const useBreadCrumbModel = () => {
     chains: [String(account?.currency.id)],
     threshold: isThresholdValid(thresold) ? Number(thresold) : 75,
     enabled: nftsFromSimplehashFeature?.enabled || false,
+    staleTime: nftsFromSimplehashFeature?.params?.staleTime,
```

Comment:
I'm not sure if we want to use staleTime as now I have a request that is sent after some time when I stay on the Collection page. We don't want to send requests if LEdger Live stays open a while on the same page.